packages/plugin-google-analytics/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@itly/plugin-google-analytics",
+  "version": "0.0.0",
+  "description": "Google Analytics Plugin",
+  "main": "dist/cjs/index.js",
+  "types": "dist/cjs/index.d.ts",
+  "module": "dist/esm/index.js",
+  "scripts": {
+    "prepublish": "yarn run build",
+    "build": "tsc -p tsconfig.json && tsc -p tsconfig-esm.json",
+    "test": "jest --runInBand"
+  },
+  "files": [
+    "LICENSE",
+    "CHANGELOG.md",
+    "README.md",
+    "dist/",
+    "package.json"
+  ],
+  "keywords": [
+    "itly",
+    "sdk",
+    "iteratively",
+    "plugin"
+  ],
+  "author": "Iteratively",
+  "license": "ISC",
```

Comment:
Can you please update the package.json to match the other plugins. Igor updated the license and added repo/homepage/bugs last week.
https://github.com/iterativelyhq/itly-sdk/blob/master/packages/plugin-braze/package.json#L27-L35